incubator/hnc/pkg/config/gvk.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import "k8s.io/apimachinery/pkg/runtime/schema"
+
+// For now, it hardcodes the list of GVKs handled by the HNC - namely, Secrets,
```

Comment:
Comments for exported names should start with that name so we can run `godoc` on it.

```
// GVKs is currently hardcoded to the set of GVKs
// handled by the HNC...
```

Reply:
Done.